cli/azd/pkg/graphsdk/util.go

```diff
@@ -0,0 +1,46 @@
+package graphsdk
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io"
+
+	"github.com/Azure/azure-sdk-for-go/sdk/azcore"
+	"github.com/Azure/azure-sdk-for-go/sdk/azcore/cloud"
+	"github.com/Azure/azure-sdk-for-go/sdk/azcore/policy"
+	"github.com/Azure/azure-sdk-for-go/sdk/azcore/runtime"
+)
+
+// Creates a new Azure HTTP pipeline used for Graph SDK clients
+func NewPipeline(
```

Comment:
NIT: must this and SetHttpRequestBody be exported?

Reply:
Ahh - I am guessing yes right now because you use them from the `graphsdk_test` package and the `_test` packages can only depend on exported members.